### DISCUSSION TITLE
[功能]: 希望支持Gemini的音频和视频处理功能。

### DISCUSSION THREAD

**Discussion by lioensky** [CherryHQ/cherry-studio#2002] [Model & Provider | 模型和服务商]
### 您的功能建议是否与某个问题相关?

官方控制台里可以发送音频和视频给AI，但是客户端里不行。

### 请描述您希望实现的解决方案

希望支持基础的音频、视频文件，比如mp3，mp4，可以发送给ai总结音频或者视频。

**Comment by pengan95** [comment]
参考项目
1. [Live Demo](https://github.com/ViaAnthroposBenevolentia/gemini-2-live-api-demo)
2. [NotebookLM](https://notebooklm.google/)

**Comment by happyTonakai** [comment]
一些模型原生支持多模态，目前cherry studio的多模态只有视觉，没有音频，希望能够补全
related: #1727 

![image](https://github.com/user-attachments/assets/1cfc0c38-f136-4640-bfbf-9da0d87d7738)

**Reply by MuChengZJU** [reply]
Gemini, Qwen VL/Omni也支持视频了，希望加入

**Comment by gaoxiaobei** [comment]
近期是否有增加此功能的计划？

**Comment by JackieJK** [comment]
request for gemini-4
